content/waysofworking.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: base
+title: Internship - Ways of Working
+permalink: waysofworking.html
+---
+ 
+# {{page.title}}
+ 
+## Project Structure
+
+As these are short-term and remote projects we propose a minimum of weekly project *"check-ins"* with the NHS project supervisor (although more than often the frequency is higher than this).  Regular code reviews & pair programming are encouraged as this increases the quality of code as well as ensuring the NHS supervisor understands the detailed progress of the work.  These meetings will focus on technical specifics and hurdles.
+ 
+More formal milestone meetings are also requested in order to run the project past the innovation team for wider comment.  These meetings will focus on deliverables and project timeline and are usually monthly presented to a wider audience who are interested in the work.  
+ 
+It is expected at the beginning of the project that these milestones will be agreed between candidate and supervisor, and put in place to support monitoring of the project timelines, but should reflect releases of product / stages of the project where possible. 
+ 
+<p align="center"> 
+  <img src="assets/img/milestoneTimeline.png" alt="Rough project structure with monthly milestones and linear sprints" width="800"/>
```

Comment:
Suggest width `600` as image is slightly pixelated at `800`

Reply:
Implemented